resize.go

```diff
@@ -9,12 +9,14 @@ import "C"
 import (
 	"errors"
 	"math"
+  "runtime"
 )
 
 // Resize is used to transform a given image as byte buffer
 // with the passed options.
 func Resize(buf []byte, o Options) ([]byte, error) {
 	defer C.vips_thread_shutdown()
+  defer runtime.KeepAlive(buf)
```

Comment:
That's true, but I don't think an explicit order is required here. The FFI C call should be agnostic to this process since the image buffer should been already moved to the Go side GC managed heap.